agent/internal/options/options.go

```diff
@@ -81,10 +80,6 @@ type Options struct {
 	// master config.
 	AgentReconnectBackoff int `json:"agent_reconnect_backoff"`
 
-	ContainerRuntime   string             `json:"container_runtime"`
```

Comment:
Same comment about leaving this option in but checking that is it always docker

So if someone has ```container_runtime: docker``` in their agent config I think we shouldn't break them

Reply:
According to our previous iterations in the PR, I thought we agreed on having to remove `ContainerRuntime` config all-together. I see what you are saying here and agree, whereas earlier I assumed because this config was never documented, it was actually used by any customers. 

@kkunapuli - Do you agree to the same?

Reply:
@ShreyaLnuHpe Yes, that was my understanding from talking with Bradley (that `container_runtime` configuration option was never documented). 

As you've dug into the task more thoroughly, it seems you've uncovered an additional use case that was documented: using agents with slurm. I'm ok with leaving the configuration option in, with non-Docker values returning an error. I though that extra config options would be ignored, not result in errors.

Reply:
Shall we now have a documentation of `container_runtime` config, and mentioning that it only intakes `docker` value in case of agentRM?

Reply:
No, I think don't document it. We won't block someone from having `container_runtime: docker` but we don't want anyone adding it either.